frontend/src/app/components/types.js

```diff
@@ -0,0 +1,14 @@
+import {
```

Comment:
same question as above but Shouldn't this file live in a directory `frontend/src/app/components/MainInstallButton/types.js`

Reply:
Maybe, but for now it has types for multiple components and not just MainInstallButton